app/src/main/java/org/oppia/app/story/StoryViewModel.kt

```diff
@@ -58,7 +59,8 @@ class StoryViewModel @Inject constructor(
   private fun processStoryChapterList(storySummary: StorySummary): List<StoryItemViewModel> {
     val chapterList: List<ChapterSummary> = storySummary.chapterList
 
-    for (position in 0..storySummary.chapterList.size) {
+    Log.d("TAG", "chapterList: " + chapterList.size)
+    for (position in 0 until chapterList.size -1) {
```

Comment:
Won't this skip the last one? 'until' automatically goes until one before the previous per https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/until.html.

Reply:
Fixed.